docs/README.md

```diff
@@ -24,6 +25,34 @@
 
 The Public Health Data Infrastructure (PHDI) projects are part of the Pandemic-Ready Interoperability Modernization Effort (PRIME), a multi-year collaboration between CDC and the U.S. Digital Service (USDS) to strengthen data quality and information technology systems in state and local health departments. Under the PRIME umberalla the PHDI project seeks to develop tools, often reffered to as Building Blocks, that State, Tribal, Local, and Territorial public health agencies (STLTs) can use to better handle the public health data they recieve. The purpose of this repository is to implement the Building Blocks devloped from the [PHDI SDK](https://github.com/CDCgov/phdi-sdk) on Google Cloud Platform (GCP). This will allow users to easily begin using these Building Blocks in their own GCP environment. For more information on using this repository beyond what is contained in this document please refer to our [Getting Started](getting_started.md) doc.
 
+### Quick Start
+
+To deploy this pipeline to your own Google Cloud environment, follow these steps.
+  
+  Be sure to replace all instances of `CDCgov` in GitHub URLs with your user or organization name.
+  1. [Install the gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
+  1. [Fork this repository](https://github.com/CDCgov/phdi-google-cloud/fork) into your personal or organization account
+  1. Clone your newly forked repository to your local machine by running:
+
+         git clone https://github.com/CDCgov/phdi-google-cloud.git
```

Comment:
Doesn't this clone our repo and not their fork of it?

Reply:
Yeah, but on line 32 I wrote "Be sure to replace all instances of `CDCgov` in GitHub URLs with your user or organization name." That might be easy to miss, though. I couldn't think of a better way to do it, besides maybe replacing `CDCgov` with `myuser` or something like that?

Reply:
Given that `git clone https://github.com/CDCgov/phdi-google-cloud.git` will run I think this creates an easy gotcha mistake for new users.